locales/en/messages.json

```diff
@@ -4310,18 +4333,58 @@
     "osdTimerSourceTooltip": {
         "message": "Select the timer source, this controls the duration/event that the timer measures"
     },
+    "osdTimerSourceOptionOnTime": {
+      "message": "On time",
+      "description": "One of the options for the source timer"
+    },
+    "osdTimerSourceOptionTotalArmedTime": {
+      "message": "Total armed time",
+      "description": "One of the options for the source timer"
+    },
+    "osdTimerSourceOptionLastArmedTime": {
+      "message": "Last armed time",
+      "description": "One of the options for the source timer"
+    },
+    "osdTimerSourceOptionOnArmTime": {
+      "message": "On/Armed time",
+      "description": "One of the options for the source timer"
```

Comment:
I think this description should describe how the option works. For context.

This option shows On Time when craft is disarmed, and Armed Time when armed.